### PR TITLE
View statement support

### PR DESCRIPTION
**Pull request by marcellustavares**

### CODE REVIEW

**Comment by goccy** [comment]
Thank you for your great PR ! I'll see this later .

File: internal/spec.go [hunk 1]
```diff
@@ -478,6 +495,19 @@ func newColumnsFromDef(def []*ast.ColumnDefinitionNode) []*ColumnSpec {
 	return columns
 }
 
+func newColumnsFromDefFromOutputColumn(def []*ast.OutputColumnNode) []*ColumnSpec {
```
Comment (goccy) [inline internal/spec.go:498]: `newColumnsFromOutputColumns` name looks better than this .

**Comment by goccy** [comment]
I only commented on one point, but the others look good. Awesome !!

**Comment by marcellustavares** [comment]
thanks for reviewing @goccy, I've pushed https://github.com/goccy/go-zetasqlite/pull/85/commits/65985ff19e0e010f0389bedc0aa08b08d29e57aa with the function rename.

Regards,

**Comment by goccy** [comment]
LGTM !

**Pull request merged**